src/views/alpha/FeedCard.tsx

```diff
@@ -264,19 +272,24 @@ export const ProfileHandle = ({ user, shorten = false }: { user: User; shorten?:
     }
   }, []);
 
+  console.log('user', user);
+
   return (
     <Popover
       isShowOnHover={true}
+      placement={props.popoverPlacement}
       content={
-        <ProfilePreview
-          address={user.address}
-          context={{
-            data: getProfilePreivewDataFromUser(user),
-            loading: false,
-            refetch: () => {},
-            fetchMore: () => {},
-          }}
-        />
+        <div className="w-80">
+          <ProfilePreview
```

Comment:
```suggestion
          <ProfilePreview
            className="w-80"
```

recommend passing the className to the ProfilePreview component. Also noticed the preview could use a little padding at the bottom of the popover.

Reply:
Also recommend min width here so if the user has collected a lot of NFTs the box stretches past the min width instead of colliding with the created.

Reply:
Noted, will remember this for a later pass of the componenet